spec/services/forest_liana/resources_getter_spec.rb

```diff
@@ -54,11 +68,80 @@ def init_scopes
       init_scopes
     end
 
-    after(:each) do
-      User.destroy_all
-      Island.destroy_all
-      Location.destroy_all
-      Tree.destroy_all
+    describe 'records eager loading' do
+      let(:resource) { Product }
+      let(:fields) { { resource.name => 'id,name,manufacturer', 'manufacturer' => 'name' } }
+
+      before do
+        orange_manufacturer = Manufacturer.create!(name: 'Orange')
+        pear_manufacturer = Manufacturer.create!(name: 'Pear')
+
+        baby_driver = Driver.create!(firstname: 'Baby driver')
+        taxi_driver = Driver.create!(firstname: 'Taxi driver')
+
+        Product.create!(name: 'Valencia', uri: 'https://valencia.com', manufacturer: orange_manufacturer, driver: baby_driver)
+        Product.create!(name: 'Blood', uri: 'https://blood.com', manufacturer: orange_manufacturer, driver: taxi_driver)
+        Product.create!(name: 'Conference', uri: 'https://conference.com', manufacturer: pear_manufacturer, driver: baby_driver)
+        Product.create!(name: 'Concorde', uri: 'https://concorde.com', manufacturer: pear_manufacturer, driver: taxi_driver)
+      end
+
```

Comment:
Just to follow what has been done previously, I would like this upper in the before each. It's not the best in terms of performances of course, it's just a matter of sticking to what currently exists. 

See the comment upper with the code I suggest ☝️